integration/tests/common/sealing_and_verification.go

```diff
@@ -65,7 +66,13 @@ func SealingAndVerificationHappyPathTest(
 	}
 
 	// waits until blockB is sealed by consensus nodes after result approvals for all of its chunks emitted.
-	blockState.WaitForSealed(t, blockB.Header.Height)
+	// blockState.WaitForSealed(t, blockB.Header.Height)
+	// waits till sealing height goes beyond the victim block
+	blockState.WaitForSealed(t, blockB.Header.Height+1)
```

Comment:
```suggestion
	// waits until we seal a height equal to the victim block height
	blockState.WaitForSealed(t, blockB.Header.Height)
```
consistency with other test case